bigbluebutton-html5/imports/ui/components/actions-bar/service.js

```diff
@@ -4,6 +4,41 @@ import { makeCall } from '/imports/ui/services/api';
 import Meetings from '/imports/api/meetings';
 import Breakouts from '/imports/api/breakouts';
 
+const processOutsideToggleRecording = (e) => {
```

Comment:
Why did you need to add `processOutsideToggleRecording` here? Perhaps slipped during conflict resolving? It was moved to navbar/service.js

Reply:
Same for `connectRecordingObserver`